### PR TITLE
416478 describe column data type coalescence

### PR DESCRIPTION
**Pull request by osisoft-mbishop**

### CODE REVIEW

File: content/analytics/data-views/data-view-link-fields.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ Data items included in a data view may have slight differences in property namin
 
 Data views can overcome property naming differences by linking these similar properties into a single data field. This applies to stream properties referenced by Id or by name, and to stream metadata keys. Asset properties can also be linked.
 
+When data fields are linked, AVEVA Data Hub converts the data types in the linked into compatible data types. For more information, see <xref:data-view-data-type-conversion>.
```
Comment (jasnames) [inline content/analytics/data-views/data-view-link-fields.md:11]:
```suggestion
When data fields are linked, AVEVA Data Hub converts the data types in the linked fields into compatible data types. For more information, see <xref:data-view-data-type-conversion>.
```

File: content/analytics/data-views/data-view-data-type-conversion.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
+---
+uid: data-view-data-type-conversion
+---
+
+# Data view data type conversion
+
+When requesting a data view that includes multiple data types in a single column, the data view will:
```
Comment (jasnames) [inline content/analytics/data-views/data-view-data-type-conversion.md:7]: This is like the intro to a bulleted list, but "the data view will..." doesn't go with step 2. I would change to

"When you request a data view that includes multiple data types in a single column:
1. The data view converts ..."

File: content/analytics/data-views/data-view-data-type-conversion.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+1. Convert two or more SDS data types into a compatible SDS data type wide enough to accommodate all types without losing information. This conversion occurs to accommodate the strongly-typed data in the Parquet format.
+
+1. When requesting a data view in the Parquet format, an additional conversion takes place. The compatible SDS data type referenced in step 1 is converted to a Parquet data type. This conversion occurs because the Parquet format does not support all data types natively.
+
+## SDS data type conversion
+
+When requesting a data view that includes multiple data types in a single column, the data view will convert two or more SDS data types into a compatible SDS data type wide enough to accommodate all types without losing information. This conversion occurs to accommodate the strongly-typed data in the Parquet format.
```
Comment (jasnames) [inline content/analytics/data-views/data-view-data-type-conversion.md:15]: This just repeats the info from above. I'd remove the first paragraph.

File: content/analytics/data-views/data-view-data-type-conversion.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+| Int64  | UInt64 | Decimal |
+| Int64  | Decimal| Decimal |
+| UInt64 | Decimal| Decimal |
+| Single | Double | Double  |
+
+Nullable types and non-nullable types convert into a nullable type. If AVEVA Data Hub cannot convert two data types into a compatible type, it will convert them into strings. When you request a data view in the Parquet format with the **Narrow** data view shape selected, AVEVA Data Hub usually converts combined columns into strings.
+
+## SDS to Parquet data type conversion
+
+When requesting a data view in the Parquet format, an additional conversion takes place. The compatible SDS data type referenced in step 1 is converted to a Parquet data type. This conversion occurs because the Parquet format does not support all data types natively.
```
Comment (jasnames) [inline content/analytics/data-views/data-view-data-type-conversion.md:75]: Again, repeats exactly the info from above. I'd remove and add an intro for the table. "The following table lists the SDS data types and their corresponding Parquet data types."

File: content/analytics/data-views/data-view-link-fields.md (hunk 1 above)
Comment (osisoft-mbishop) [reply content/analytics/data-views/data-view-link-fields.md:11]: I ended up taking this part out on SME recommendation.

**Review by jasnames — APPROVED** [review]

**Pull request merged**